forms-flow-theme/scss/_typography.scss

```diff
@@ -1,9 +1,70 @@
-
 $input-color: $dark;
 $input-focus-color: $dark;
-
+$h4-font-size:1.275rem ;
+$h5-font-size : 1rem ;
 
 .input-group-text {
-    color: $gray-800 !important;
+  color: $gray-800 !important;
+}
+
+.text-primary {
```

Comment:
is this really needed ?

Reply:
Yes. Bootstrap text-primary color and formsflow text-primary color are different.

Reply:
updated . By overriding the primary color variable, as all Bootstrap components that use the primary color will update automatically. So removed classes which is related to primary color .